docs/learning/presentation/index.md

```diff
@@ -1,37 +1,67 @@
 
-## Purpose
-
-iTwin.js Presentation library helps retrieve presentation data from iModels and
-takes care of unified selection. What data to retrieve from an iModel is
-defined by consumers in a declarative way, using presentation rules - that's
-why the library is commonly called "presentation rules engine".
-
-![library purpose](./library.png "Purpose of the library")
-
-[Getting started with iTwin.js Presentation](./Setup.md)
-
-## Library Contents
-
-- Content
-  - [Rules](./Content/index.md)
-  - [Property Categorization](./Content/PropertyCategorization.md)
-  - [ECExpressions](./Content/ECExpressions.md)
-  - [Terminology](./Content/Terminology.md)
-- Hierarchies
-  - [Rules](./Hierarchies/index.md)
-  - [ECExpressions](./Hierarchies/ECExpressions.md)
-  - [Terminology](./Hierarchies/Terminology.md)
-- Customization
-  - [Rules](./Customization/index.md)
-  - [ECExpressions](./Customization/ECExpressions.md)
-  - [Property value renderers](./Customization/PropertyValueRenderers.md)
-  - [Property category renderers](./Customization/PropertyCategoryRenderers.md)
-- [Default BIS Presentation Rules](./DefaultBisRules.md)
-- [Unified Selection](./Unified-Selection/index.md)
-- [ECExpressions](./ECExpressions.md)
-- [Localization](./Localization.md)
-- [Ruleset Variables](./RulesetVariables.md)
-- [Ruleset Supplementation](./RulesetSupplementation.md)
-- [Performance Tips](./Performance.md)
-- [Testing](./Testing.md)
+## The Problems
+
+As explained in the [ECSQL](../ECSQL.md) overview page, data from iModels can be queried using the language just like querying from any relational database using SQL. That's often the most straightforward way to get raw data from the iModels.
+
+> Query:
+>
+> ```sql
+> SELECT UserLabel, Category, StructuralQuantities__x002F____x0040__AreaSurfaceNetModeled, StructuralQuantities__x002F____x0040__Length
+> FROM BuildingDataGroup:Concrete__x0020__Pile
+> WHERE ECInstanceId = 0x20000028e6b
+> ```
+>
+> Result:
+>
+> ```text
+> | UserLabel                         | Category                                                                       | StructuralQuantities__x002F____x0040__AreaSurfaceNetModeled | StructuralQuantities__x002F____x0040__Length |
+> | --------------------------------- | ------------------------------------------------------------------------------ | ----------------------------------------------------------- | -------------------------------------------- |
+> | Foundation | Concrete Pile - Pile | {"id":"0x200000004f4","relClassName":"BisCore.GeometricElement3dIsInCategory"} | 63.24338455489202                                           | 27.982053001523                              |
+> ```
+>
+> [Try it yourself](/console/?imodel=Metrostation%20Sample&query=SELECT%20UserLabel,Category,StructuralQuantities__x002F____x0040__AreaSurfaceNetModeled,StructuralQuantities__x002F____x0040__Length%20FROM%20BuildingDataGroup:Concrete__x0020__Pile%20WHERE%20ECInstanceId=0x20000028e6b)
```

Comment:
The iframe opens `/console/`, so I suppose it just doesn't get deployed locally, but is available when deployed to dev/qa/prod. If you open `/learning/ecsqltutorial/` you'll see the iframe shows the same `Cannot GET /console/` error.

I agree that embedding the console with the prepared query is a better option compared to what I'm doing now. @williamkbentley, do you know if we can get the query executed as soon as the console opens? In the examples I see the query is added to the textbox, but is not executed until the "Run" button is pressed.

Reply:
Yes, the console is deployed alongside the docSite, so it will not be available locally. I believe @BaoTon attempted to add the "run on load" feature a few years ago, but was unable. There were some challenges, the details escape me. @BaoTon maybe we can give that feature a fresh look?

Reply:
Thanks @williamkbentley. Implementing "run on load" for the console is outside the scope of this PR, so I'd say let's go with what we have at the moment. Agree @roluk?

Reply:
"run on load" shouldn't be an issue - I assumed users would want to see the query before it executes. Seeing results before any action happens was a bit awkward, but I could add an option to the iframe. I'll add this as a backlog item.